packages/ui-tests/cypress/support/next-commands/datamapper.ts

```diff
@@ -1,13 +1,13 @@
 Cypress.Commands.add('attachSourceBodySchema', (filePath: string) => {
   cy.get('[data-testid="attach-schema-sourceBody-Body-button"]').click();
   cy.get('[data-testid="attach-schema-file-input"]').attachFile(filePath);
-  cy.get('[data-testid="expand-source-icon"]').should('be.visible');
+  cy.get('[data-testid^="expand-source-icon"]').should('be.visible');
 });
 
 Cypress.Commands.add('attachTargetBodySchema', (filePath: string) => {
   cy.get('[data-testid="attach-schema-targetBody-Body-button"]').click();
   cy.get('[data-testid="attach-schema-file-input"]').attachFile(filePath);
-  cy.get('[data-testid="expand-target-icon"]').should('be.visible');
+  cy.get('[data-testid^="expand-target-icon"]').should('be.visible');
```

Comment:
Although there is only `Body` at target side, we might be better to explicitly specify the target body ID for the future - at some point we would introduce target header and target variable.

Reply:
ok, makes sense. Will explicitly specify the `Body` ID.